cypress/e2e/regression/sidebar_2.cy.js

```diff
@@ -45,4 +47,19 @@ describe('Sidebar added sidebar tests', () => {
     sideBar.openSidebar()
     sideBar.verifySafeReadOnlyState(staticSafe100)
   })
+
+  it('Verify Fiat currency changes when edited in the assets tab', () => {
+    assets.changeCurrency(constants.currencies.cad)
+    sideBar.checkCurrencyInHeader(constants.currencies.cad)
+  })
+
+  it('Verify "wallet" tag counter if the safe has tx ready for execution', () => {
```

Comment:
This test description is not correct. That counter counts how many tx in queue the current connected owner can sign, not necesarely they can be executed.
Change it to something like "Verify "wallet" tag counter of tx that need signature from current owner"

Reply:
This is original test title taken from our hub of tests and scheduled for automation, the implementation is according to the title. If there are changes needed in updating test title, we can do this later, outside of the scope of this task.